insta/src/lib.rs

```diff
@@ -258,8 +259,10 @@
 //! opt-level = 3
 //! ```
 //!
-//! You can also disable the default features of `insta` which will cut down on
+//! You can also disable the default features of [`insta`] which will cut down on
 //! the compile time a bit by removing some quality of life features.
+//!
+//!  [`insta`]: https://docs.rs/insta
```

Comment:
This is self-referential, no? I also think this is over-structured...

Reply:
Yes and no.

We reference `similar` here => linking `insta` too looks nicer..
![image](https://github.com/user-attachments/assets/21d39812-2248-4aa6-882d-c3f3ad13d984)